resources/lib/ui/http.py

```diff
@@ -31,6 +32,11 @@ def Session():
         })
         _SESSION = s
 
+        token = TokenDecoder.get_token(lambda x: __send_request(s, x, None, None, False))
```

Comment:
in general, i want the change to be as much as possilbe outside of UI directory,
as UI directory is intended to be used generically with Kodi
while NineAnimeBrowser is the more specific one.
we need to think how to tackle this.